web/styles/compose.css

```diff
@@ -931,9 +931,12 @@ textarea.new_message_textarea {
         box-shadow: none;
         outline: none;
 
-        padding: 4px 6px;
+        padding: 0.15em 0.43em;
```

Comment:
Since ems are relative values, it's really important to preserve how they were calculated as comments in the CSS. There's a bunch of examples throughout the codebase where others, including me, have done this.

Reply:
(Same holds true for the line-height value you specify below.)

Reply:
Calculations for em values now added to comments.

Reply:
No, this is not what should be happening here. The codebase should indicate where `0.15em` originated, not what it computes to at different sizes. This is made even more puzzling because where did values like 6.88px come from, compared to the original whole-pixel values on the padding.

Reply:
Just so I understand this correctly, I should add a comment explaining why we needed to change from 4px to 0.15em for the top and bottom padding? Something like: 

> "Top and bottom padding needs to be a dynamic value to fit emojis in both compact and regular mode on all monitors."

The right and left padding does not actually need to be an em value, keeping it at 6px works fine as it only is the top and bottom padding that is part of the problem. Perhaps this is better and less confusing. 6.88px is just a byproduct of trying to set a value around 6px using em at 14px and 16px.

Reply:
That's probably fine. Em values should be calculated from pixel values, and if the pixel value you're working with is also an adjustment, that should be noted somewhere too (probably in your commit or the PR).

Reply:
I have updated the commit with the new changes.